tests/misc.py

```diff
@@ -1,5 +1,22 @@
+from kubernetes import client
+from kubernetes.client import Configuration, ApiClient
 
-class ContainerSpec():
+
+class K8SApiClient(ApiClient):
+
+    def __init__(self):
+        client_config = Configuration()
+        client_config.connection_pool_maxsize = 32
+        super(K8SApiClient, self).__init__(configuration=client_config)
+
+
+class CoreV1ApiClient(client.CoreV1Api):
```

Comment:
we could use a shorter name :)

Reply:
tried to keep the same length as before ;-)